src/service.js

```diff
@@ -83,20 +88,27 @@ LS2Request.prototype.send = function() {
 		}
 		if(!self.subscribe) {
 			self.cancel();
+		} else if(self.ts && refs[self.ts]) {
```

Comment:
Correct. As mentioned, I was unsure of any way to guarantee when subscribed functions should be removed from the ref list. At cancel? at some specific return value? Never? So the tentative change had it set to remove it after the first callback and advise those using subscription service requests to hold references themselves. If there is a better solution, I'm more than willing to change/refactor :)